src/Components/CalDate.js

```diff
@@ -6,36 +6,41 @@ import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 
 class CalDate extends React.Component {
-
+    
     constructor() {
         super();
         this.color_change = this.color_change.bind(this);
     }
-
+    
     color_change() {
-
+        
         var d = new Date()
         var weekDay = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
         var months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
-
+        
         var day = weekDay[d.getDay()]
         var month = months[d.getMonth()]
         var date = d.getDate()
         var year = d.getFullYear()
-
+        
         var currentDate = day + ", " + month + " " + date + ", " + year
-
+        
         document.getElementById("currentDate").innerText = currentDate;
-
+        
         var hours = d.getHours();
         var minutes = d.getMinutes();
         var timeOfDay = "";
-
-        if (hours > 12) {
-            hours = hours - 12;
+        
```

Comment:
this looks like a great place to refactor this code into a common function.  looks replicated in the file CalenderEvent.js

Reply:
corrected